recipe_scrapers/_grouping_utils.py

```diff
@@ -85,43 +84,35 @@ def group_ingredients(
     group_element: str,
 ) -> List[IngredientGroup]:
     """
-    Group ingredients into sublists according the heading in the recipe.
+    Group ingredients into sublists according to the heading in the recipe.
 
-    The group headings are determined by html element matching the group_heading CSSselector.
-    The group items are determined by extracting the text the html elements that are
-    below the group heading, then matching the text to one of the ingredients in ingredients_list.
-    This is done to ensure this function returns the same ingredients as are returned from
-    the .ingredients() method of a scraper.
-
-    If the recipe doesn't group ingredients, this returns a single IngredientGroup object
-    with the purpose set to None and the ingredients list populated with all ingredients.
+    This method groupings ingredients by extracting headings and corresponding elements
```

Comment:
Couple of typos:

This method **_groups_** ingredients by extracting headings and corresponding elements
based on provided CSS selectors. If no groupings are present, it **_creates_** a single
group with all ingredients. It ensures ingredient groupings match those i
the .ingredients() method of a scraper by comparing **_the_** text against the ingredients list.